lib/src/session/async_item.rs

```diff
@@ -14,6 +15,7 @@ pub enum AsyncItem {
     Body(Body),
     StreamingBody(StreamingBody),
     PendingReq(PeekableTask<Response<Body>>),
+    PendingLookup(PeekableTask<Result<Vec<u8>, ObjectStoreError>>),
```

Comment:
What do you think about aliasing `PeekableTask<Result<Vec<u8>, ObjectStoreError>` as `PendingLookup` and then using that throughout the code?

Reply:
That sounds reasonable to me

Reply:
Can we call this variant `PendingKvLookup` (and rename the session functions accordingly)?

Reply:
Done :+1: